src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs

```diff
@@ -148,14 +150,29 @@ public override MetadataFileResult MovieMetadata(Movie movie, MovieFile movieFil
 
                 details.Add(new XElement("studio", movie.Studio));
 
-                if (image == null)
+                if (thumbnail == null)
                 {
                     details.Add(new XElement("thumb"));
                 }
 
                 else
                 {
-                    details.Add(new XElement("thumb", image.Url));
+                    details.Add(new XElement("thumb", thumbnail.Url));
```

Comment:
It is indeed unclear whether the tag in this form is still supported. Since the official documentation only mentions `<thumb aspect="poster" preview=""></thumb>`, it may be more correct to ditch the old declaration without the `aspect` attribute. I can however verify that the "aspectless" element does not interfere with the "aspectful" element when Kodi is looking for poster entries, so I included it to prevent the old mechanism from being removed.